src/app/ApplicationTemplate.Shared.Views/Helpers/DependencyObjectExtensions.cs

```diff
@@ -100,19 +120,27 @@ public static IEnumerable<DependencyObject> GetParentHierarchy(this DependencyOb
 
 		public static IEnumerable<T> FindAllChildren<T>(this DependencyObject element, int? childLevelLimit = null, bool includeCurrent = true)
 			where T :
+//-:cnd:noEmit
 #if HAS_UMBRELLA_UI
+//-:cnd:noEmit
 			class,
+//-:cnd:noEmit
 #endif
+//-:cnd:noEmit
 			DependencyObject
 		{
 			return element.FindAllChildren<T>(x => true, childLevelLimit, includeCurrent);
 		}
 
 		public static IEnumerable<T> FindAllChildren<T>(this DependencyObject element, Func<T, bool> selector, int? childLevelLimit = null, bool includeCurrent = true)
 			where T :
+//-:cnd:noEmit
 #if HAS_UMBRELLA_UI
```

Comment:
As discussed this will be treated separately with a overall review of conditional compiled directives